app/main.py

```diff
@@ -2,0 +1,11 @@
+def copy_file(command: str) -> str | None:
+    command_info = command.split(" ")
+
+    if "cp" in command_info[0]:
+        if command_info[1] == command_info[2]:
+            return None
+        else:
+            with (open(command_info[1], "r") as original_file,
+                  open(command_info[2], "w") as new_file):
+                for line in original_file.readline():
```

Comment:
The method `readline()` reads a single line from the file. To iterate over all lines, you should use `for line in original_file:` instead.